.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,33 @@
+name: Dockerize API
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Publish to Docker hub
+        uses: docker/build-push-action@v1
+        with:
+          context: ./
+          file: ./Dockerfile
+          push: true
+          tags: 0.1.0
```

Comment:
tag must be calculated dynamically. Create a __init__.py file that has __version__ variable and use it